src/psyclone/psyir/transformations/matmul2code_trans.py

```diff
@@ -0,0 +1,334 @@
+# -----------------------------------------------------------------------------
```

Comment:
This file has a few pylint errors.

Reply:
I've fixed the pycodestyle errors (sorry again). The pylint errors are to do with too many branches and too many locals. Should I disable these?

Reply:
Yes please.

Reply:
Done. I've added the optional argument to apply as well to make pylint happier.